sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java

```diff
@@ -101,6 +101,62 @@ public Long scheduleMessage(ServiceBusMessage message, Instant scheduledEnqueueT
         return asyncClient.scheduleMessage(message, scheduledEnqueueTime).block(tryTimeout);
     }
 
+    /**
+     * Sends a scheduled message to the Azure Service Bus entity this sender is connected to. A scheduled message is
+     * enqueued and made available to receivers only at the scheduled enqueue time.
+     *
+     * @param message Message to be sent to the Service Bus Queue or Topic.
+     * @param scheduledEnqueueTime Instant at which the message should appear in the Service Bus queue or topic.
+     * @param transactionContext to be set on message before sending to Service Bus.
+     *
+     * @return The sequence number of the scheduled message which can be used to cancel the scheduling of the message.
+     *
+     * @throws NullPointerException if {@code message} or {@code scheduledEnqueueTime} is {@code null}.
+     */
+    public Long scheduleMessage(ServiceBusMessage message, Instant scheduledEnqueueTime,
```

Comment:
I remember I had tried to keep the methods grouped by visibility, and then alphabetised.. so if you go to source, you wouldn't have to jump around. The send method should be with all its friends. Same with any o5ther public method you added.